src/auth.zig

```diff
@@ -107,6 +97,296 @@ test MTLS {
     // TODO there's likely a few more error states we should validate;
 }
 
+/// Default CookieAuth Helper uses Sha256 as the HMAC primitive.
+pub const CookieAuth = cookieAuth(Hmac.sha2.HmacSha256);
```

Comment:
Should this just be called Cookie since it's already in the auth namespace?

Reply:
Yes it should, I'll change Mtls as well